PeerioTechnologies/peerio-mobile
# feat: popup leave room illustration [branch ch10821]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/10821/mobile-leave-room-confirmation
https://app.clubhouse.io/peerio/story/7245/mobile-implement-email-confirmation-dialog-illustration
https://github.com/PeerioTechnologies/peerio-icebear/pull/253
#### Testing instructions  
1. Verify that leave room confirmation popup matches mocks.
2. I also refactored the email invite confirmation popup (accessible through the second step in contact sync on sign up), please double check that. CH task for that is in relevant links list above.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
